doc/modules/cassandra/pages/new/java11.adoc

```diff
@@ -26,11 +26,15 @@ Table 1 : Support Matrix for Java
 |===
 | |Java 8 (Run) |Java 11 (Run)
 |Java 8 (Build) |Supported |Supported
-|Java 11(Build) |Not Supported |Supported
+|Java 11(Build) |Not Supported |Experimental
```

Comment:
come on... This looks weird, just like we can only run in experimental mode :)

Reply:
Port from Apache... I should have left this file completely out I think....

Reply:
Actually it is not weird - we test it in CI, but we ship only build 8 + run 8 and 11.... or at least that we did at the time this doc was edited. So we experiment running it in CI only with build + run 11. 
Wondering whether we should just revert changes to docs here? Does it matter for HCD?

Reply:
We should update the doc to match what we think it should be for CNDB/HCD

Reply:
Ok, for which of the two we should update though?
We run CNDB with 22 and main and HCD with 11 and main. 
That will change with main-5.0.
I would assume we want to update for HCD?

Reply:
What ever we think it is for HCD

Reply:
I will do it tomorrow. Thanks for clarifying it